MAUI/DataGrid/Editing.md

```diff
@@ -0,0 +1,376 @@
+ * [RowColumnIndex](): Gets the current row and column index of the DataGrid.
+ * [Column](): Gets the Grid Column of the SfDataGrid.
+
+To hook the `SfDataGrid.CurrentCellBeginEdit` event, follow the code example:
+
+{% highlight c# %}
+this.dataGrid.CurrentCellBeginEdit += DataGrid_CurrentCellBeginEdit;
+
+private void DataGrid_CurrentCellBeginEdit(object sender, DataGridCurrentCellBeginEditEventArgs e)
+{
+    // Editing prevented for the cell at RowColumnIndex(2,2).
+    if (e.RowColumnIndex == new Syncfusion.Maui.GridCommon.ScrollAxis.RowColumnIndex(2, 2)) 
+        e.Cancel = true;
+}
+{% endhighlight %}
+
+### CurrentCellEndEdit
+
+The [CurrentCellEndEdit]() event occurs when the CurrentCell exits the edit mode. The [DataGridCurrentCellEndEditEventArgs]() has following members which provides information for `SfDataGrid.CurrentCellEndEdit` event:
+
+ * [RowColumnIndex](): Gets the current row and column index of the DataGrid.
+ * [Cancel](): When this member set to ‘true’, the event is canceled and the edited value is not committed in the underlying collection.
+
+To hook the `SfDataGrid.CurrentCellEndEdit` event, follow the code example:
+
+{% highlight c# %}
+this.dataGrid.CurrentCellEndEdit += DataGrid_CurrentCellEndEdit;
+
+private void DataGrid_CurrentCellEndEdit(object sender, DataGridCurrentCellEndEditEventArgs e)
+{
+    // Editing prevented for the cell at RowColumnIndex(1,3).
+    if (e.RowColumnIndex == new Syncfusion.Maui.GridCommon.ScrollAxis.RowColumnIndex(1, 3))
+        e.Cancel = true;
+}
+
+{% endhighlight %}
+
+## Programmatically edit a cell
+
+### Begin editing
+
+The SfDataGrid allows you to programmatically edit a cell by calling the [SfDataGrid.BeginEdit]() method. By calling this method, the particular cell enters into edit mode, allowing you to edit the data manually or programmatically. To programmatically edit a cell, refer to the following code example:
+
+{% highlight c# %}
+this.dataGrid.Loaded += DataGrid_Loaded;
+private void DataGrid_Loaded(object sender, EventArgs e)
+{
+    //Edit the cell at 2nd row,2nd column programmatically
+    this.dataGrid.BeginEdit(2, 2);
+}
+{% endhighlight %}
+
+### End editing
+
+The [SfDataGrid.EndEdit]() method can be called to programmatically end editing. When this method is called, a cell that is currently in edit mode will commit the edited value to the underlying collection and exit the edit mode. To programmatically end the editing process, refer to the following code example:
+
+{% highlight c# %}
+this.dataGrid.EndEdit();
+{% endhighlight %}
+
+### Cancel editing
+
+The [SfDataGrid.CancelEdit]() method can be called to programmatically cancel editing. When this method is called, a cell that is currently in edit mode will exit the edit mode without committing the edited value to the underlying collection. To programmatically cancel the editing process, refer to the following code example:
```

Comment:
The SfDataGrid.CancelEdit() method allows you to programmatically cancel the editing process. When called, this method exits the edit mode of a cell without committing the edited value to the underlying collection. To programmatically cancel the editing process, use the code example provided:

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/777ce7b4-9710-45d7-a773-10103260ec8a)